README.md

```diff
@@ -36,43 +36,31 @@ To this end, how can we easily access our API key without opening ourselves up t
 
 There are many ways to store sensitive information but we will go with this method. 
 
-#### Create a `.secret/` directory
+#### Create a `secret/` directory
 
-This will create a new folder in your project directory where you can store files for any of the API information you have. If you were to push this project to GitHub (ie. share it online) we could put this file in our .gitignore so it would not be shared. To this end, you will be the only one with access to this folder and your API key will be secure!
+Click on the Illumidesk logo in the top left corner while holding the `cmd` key (`ctrl` for Windows) to open the window in a new tab. This is the root of your Illumidesk directory. In this new window, click `New` in the top right corner followed by `Folder`. Find the folder titled `Untitled Folder`, select the checkbox directly to the left of it, scroll back to the top of the page and click `Rename`. Rename the folder to `secret`.
```

Comment:
I would add back in the explanation about how this will be used for storing API-related files.

Something like "First we will create a new folder in your IllumiDesk directory where you can store files related to your API credentials, and share those credentials across multiple labs." and then proceed to the "Click on the..." directions

Also we've been stylizing it as "IllumiDesk" rather than "Illumidesk"

Reply:
Should be good to go now!